tools/asciidoctor/postprocessor.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+class Copyright < Asciidoctor::Extensions::Postprocessor
```

Comment:
Minor: This script seems to just be doing basic string substitutions. Could we have written this script in Python? (We already have other Python scripts in quick-lint-js, but no Ruby scripts (except the Homebrew formula, which doesn't really count).)

Reply:
This is fine. I'm not really making use of Asciidoctor's document parameter in the process functions.